GenumKit/Stencil/Contexts.swift

```diff
@@ -66,7 +66,10 @@ extension AssetsCatalogParser {
     - `name`: `String`
     - `scenes`: `Array` (absent if empty)
        - `identifier`: `String`
-       - `class`: `String` (absent if generic UIViewController)
+       - `customClass`: `String` (absent if generic UIViewController)
+       - `isBaseViewController`: `Bool`, indicate if the baseType is 'viewController' or anything else
+       - `baseType`: `String` (absent if class is a custom class). The base class type on which a scene is base.
+          Possible values include 'viewController', 'navigationController', 'tableViewController'…
```

Comment:
Reading th code, it seems you always uppercase the first letter of this value. So:

1. either the documentation should reflect that and list the examples with an uppercase letter
2. or the code should be fixed to stop upper-casing the first letter and keeping the value intact (letting that transformation responsibility to the template using a filter if it needs it)

I think the 1st solution is the easiest and more useful, as people will always use the `uppercaseFirst` variant of this in their template anyway.